pycqed/instrument_drivers/meta_instrument/qubit_objects/CCL_Transmon.py

```diff
@@ -3680,6 +3707,116 @@ def measure_gst_1Q(self,
         a = ma2.GST_SingleQubit_DataExtraction(label='Single_qubit_GST')
         return a
 
+    def make_qubit_dac_arc(self, dac_values=None, polycoeffs=None, MC=None,
+                           nested_MC=None, fluxChan=None):
+        """
+        Creates a qubit DAC arc by fitting a polynomial function through qubit
+        frequencies obtained by spectroscopy.
+
+        Arguments:
+        - dac_values: DAC values that are to be probed
+        - polycoeffs: initial coefficients of a second order polynomial.
+        """
+
+        if dac_values is None:
+            if self.fl_dc_V0() is None:
+                dac_values = np.linspace(-3e-3, 3e-3, 11)
+            else:
+                dac_values_1 = np.linspace(self.fl_dc_V0(),
+                                           self.fl_dc_V0() + 3e-3,
+                                           11)
+                dac_values_2 = np.linspace(self.fl_dc_V0(),
+                                           self.fl_dc_V0() - 3e-3,
+                                           11)
+
+        dac_values = np.concatenate([dac_values_1, dac_values_2])
+
+        if MC is None:
+            MC = self.instr_MC.get_instr()
+
+        if nested_MC is None:
+            nested_MC = self.instr_nested_MC.get_instr()
+
+        fluxcontrol = self.instr_FluxCtrl.get_instr()
+        if fluxChan is None:
+            dac_par = fluxcontrol.parameters[(self.cfg_dc_flux_ch())]
+        else:
+            dac_par = fluxcontrol.parameters[(fluxChan)]
+
+        d = cdf.Tracked_Qubit_Spectroscopy(qubit=self,
```

Comment:
Maybe add a comment here (and also in the docstring of this method) how tracked spec tracks. How does it update the frequencies?